Content.Server/EntityEffects/Effects/PlantMetabolism/PlantDestroySeeds.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Content.Server.Botany.Components;
+using Content.Server.Botany.Systems;
+using Content.Shared.EntityEffects;
+using Robust.Shared.Prototypes;
+using Content.Shared.Popups;
+
+using Content.Shared.Popups;
+
+namespace Content.Server.EntityEffects.Effects.PlantMetabolism;
+
+public sealed partial class PlantDestroySeeds : EntityEffect
+{
+    public override void Effect(EntityEffectBaseArgs args)
+    {
+        if (
+            !args.EntityManager.TryGetComponent(args.TargetEntity, out PlantHolderComponent? plantHolderComp)
+            || plantHolderComp.Seed == null
+            || plantHolderComp.Dead
+            || plantHolderComp.Seed.Immutable
+        )
+            return;
+
+        if (plantHolderComp.Seed == null)
```

Comment:
You already check for this above.

Reply:
Oops number 2, I didn't notice since I copied it from Robust Harvest.